### PR TITLE
Add current key resolver.

### PR DESCRIPTION
**Pull request by dmitrizagidulin**

### CODE REVIEW

File: lib/keyResolver.js [hunk 1]
```diff
@@ -1,9 +1,15 @@
+const didIo = require('did-io');
+// Config did-io to support did:key driver
+didIo.use('key', require('did-method-key').driver());
```
Comment (dlongley) [inline lib/keyResolver.js:3]: Need to use `import` not `require`.

File: lib/keyResolver.js [hunk 2]
```diff
@@ -1,9 +1,15 @@
...
+
 /*!
  * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
  */
 export async function createKeyResolver() {
   return async function keyResolver({id} = {}) {
-    // TODO: implement `did:key` resolver
-    throw new Error('bedrock-tokenization "keyResolver" not implemented');
+    if(!id.startsWith('did:')) {
+      throw new Error(`Key id not supported in resolver: '${id}'.`);
```
Comment (dlongley) [inline lib/keyResolver.js:11]:
```suggestion
      throw new Error(`Key ID "${id}" not supported in resolver.`);
```

File: lib/keyResolver.js [hunk 3]
```diff
@@ -1,9 +1,15 @@
...
+    }
+    return await didIo.get({did: id});
```
Comment (dlongley) [inline lib/keyResolver.js:13]:
```suggestion
    return didIo.get({did: id});
```

File: lib/keyResolver.js (hunk 1 above)
Comment (aljones15) [reply lib/keyResolver.js:3]: I will fix this in a second @dmitrizagidulin I think we are good here.

Comment (aljones15) [reply lib/keyResolver.js]: Addressed here: https://github.com/digitalbazaar/bedrock-tokenization/pull/2/commits/0ea252be34d3dd3f626b11e7a8e9c9a3cafcfca7

File: lib/keyResolver.js [hunk 4]
```diff
@@ -1,9 +1,18 @@
 /*!
  * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
  */
+
```
Comment (dlongley) [inline lib/keyResolver.js:4]:
```suggestion
```

**Pull request merged**